packages/ezirmin/ezirmin.0.2.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer:   "KC Sivaramakrishnan <sk826@cl.cam.ac.uk>"
+authors:      ["KC Sivaramakrishnan" "Thomas Gazagnaire" "Benjamin Farinier"]
+homepage:     "https://github.com/kayceesrk/ezirmin"
+doc:          "https://kayceesrk.github.io/ezirmin/"
+license:      "ISC"
+dev-repo:     "https://github.com/kayceesrk/ezirmin.git"
+bug-reports:  "https://github.com/kayceesrk/ezirmin/issues"
+tags: []
+available: [ ocaml-version >= "4.01.0"]
+depends: [
+  "ocamlfind"   {build}
+  "ocamlbuild"  {build}
+  "topkg"       {build}
+  "lwt"
+  "git-unix"
+  "irmin"				{>= "0.12.0"}
+	"irmin-watcher"
+  "ptime"
+	"ppx_jane"
+]
+depopts: []
+build: [
+  "ocaml" "pkg/pkg.ml" "build"
+          "--pinned" pinned ]
```

Comment:
the `pinned` needs to be escaped for opam 1.2.0 support -- `"--pinned" "%{pinned}%"`